### PR TITLE
DP-1168 install ros package + proj mobtest

### PR DESCRIPTION
**Pull request by duartecoelhomovai**

### CODE REVIEW

**Review by AlexFernandes-MOVAI — CHANGES_REQUESTED** [review]
Please add a link to a working execution of this workflow

**Comment by AlexFernandes-MOVAI** [comment]
and resolve conflicts

**Review by diasdm — CHANGES_REQUESTED** [review]
Please update the PR description with impacts. It is required for a `build-requirements.txt` to exist in the repo?

**Comment by duartecoelhomovai** [comment]
https://github.com/MOV-AI/movai_autonomy_pack/actions/runs/5321449666/jobs/9637666868

**Review by AlexFernandes-MOVAI — APPROVED** [review]

File: .github/workflows/ros-workflow.yml [hunk 1]
```diff
@@ -244,10 +247,54 @@ jobs:
...
+          sudo apt update
+          sudo apt install mobros=${{ inputs.MOBROS_VERSION }} -y
+      - name: Install debian artifacts
+        run: |
+          cd artifacts/
+          PACKAGES_INSTALL="$(find -L ./ -name "*.deb")"
+          echo $PACKAGES_INSTALL
+          sudo mobros install $PACKAGES_INSTALL -y --upgrade-installed --fail-on-apt-update
+
+      - name: Mobtest
```
Comment (diasdm) [inline .github/workflows/ros-workflow.yml]:
```suggestion
      - name: Mobtest project validations
```

**Review by diasdm — APPROVED** [review]

**Pull request merged**